CHANGELOG.md

```diff
@@ -5,11 +5,16 @@ It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
 ## Kiwix-JS v2.8.0
+
 Released on *TODO*
 
-Detailed changelog : TODO
```

Comment:
The detailed changelog line should be kept, with https://github.com/kiwix/kiwix-js/milestone/22?closed=1

Reply:
OK, thanks! Done (in commit to be pushed shortly).